pkg/fixtures/writer_s3.go

```diff
@@ -0,0 +1,103 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	"os"
+	"path/filepath"
+)
+
+type BlobFixturesSettings struct {
+	BasePath   string
+	ConfigName string
+}
+
+type s3FixtureWriter struct {
+	logger      mon.Logger
+	store       blob.Store
+	batchRunner *blob.BatchRunner
+
+	settings *BlobFixturesSettings
+	purger   *s3Purger
```

Comment:
Please follow our rules in terms of sorting fields on structs:
```
Please sort properties on a struct.
1. embedded structs/interfaces
2. other structs/interfaces
3. scalars

Within each group, sort alphabetically.

The following exceptions apply:

the logger should be listed first after all embeddables, as it's the most used dependency (functionality only)
timestamps (CreatedAt, UpdatedAt) are the last properties (entities only)
```

please also adhere to that when it comes to: initalization of a struct and interfaces in general.

Reply:
Already applied.